golem/task/requestedtaskmanager.py

```diff
@@ -395,6 +403,14 @@ def _get_task_api_service(
             shared_dir=shared_dir
         )
 
+    @staticmethod
+    def _check_task_timeout(task_id: TaskId) -> None:
+        task = RequestedTask.get(RequestedTask.task_id == task_id)
+        if task.status != TaskStatus.finished:
+            logger.info("Task %r timed out", task_id)
```

Comment:
Please use kwargs writing for the logging data
```suggestion
            logger.info("Task timed out. task_id=%r", task_id)
```